tests/src/test/spark330/scala/org/apache/spark/sql/rapids/utils/RapidsSQLTestsBaseTrait.scala

```diff
@@ -121,6 +121,12 @@ object RapidsSQLTestsBaseTrait {
         "org.apache.spark.sql.rapids.ExecutionPlanCaptureCallback")
       .set("spark.sql.warehouse.dir", warehouse)
       .set("spark.sql.cache.serializer", "com.nvidia.spark.ParquetCachedBatchSerializer")
+      .set("spark.sql.session.timeZone", "UTC")
+      .set("spark.rapids.sql.explain", "ALL")
+      // uncomment below config to run `strict mode`, where fallback to CPU is treated as fail
```

Comment:
nit: do we want a parameterized test suit here for both `strict mode` and non-`strict mode` associated with different `RapidsTestSettings`?

Reply:
A lot more test cases will fail in strict mode as shown in our report, and we'll need even more resource to take care of the failed test cases in strict mode. 

Now that we're still struggling with non-strict mode, the result of strict mode is more of a reference than a guide to our daily priority. So it's not mandatory to do so at present? we can add that anytime when we have enough resource to deal with failed test cases in strict mode .

Reply:
Also we should consider how to parallel execute these two modes to avoid CI taking too much time